static/app/components/replays/videoReplayer.tsx

```diff
@@ -203,12 +289,34 @@ export class VideoReplayer {
     video.style.display = 'block';
   }
 
-  protected playVideo(video: HTMLVideoElement | undefined): Promise<void> {
+  protected async playVideo(video: HTMLVideoElement | undefined): Promise<void> {
     if (!video) {
       return Promise.resolve();
     }
     video.playbackRate = this.config.speed;
-    return video.play();
+
+    // If video is not playable, then update buffering state,
+    // otherwise, proceed to hide the previous video and play
+    if (video.readyState === 0) {
```

Comment:
should we be waiting until `readyState === 4`? i'm looking at https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/readyState, but i'm also not sure if we need all of that info loaded before buffering can be safely set as `false`

Reply:
Yeah good call -- I'm not sure tbh, I wanted to wait for the `Range` changes before testing this some more.